frontend/interfaces/host.ts

```diff
@@ -303,6 +303,7 @@ export interface IHost {
   display_text: string;
   display_name: string;
   target_type?: string;
+  scripts_enabled?: boolean;
```

Comment:
just want to confirm whether the field is actually optional or if it's `boolean | null`

Reply:
^ @getvictor?

Reply:
The field will always be present. Is there any frontend distinction between false and null? The backend can always default it to false. For example, if we don't yet know if scripts are enabled (but I think we will always know).

Reply:
Thanks Victor. I believe so, but mostly to keep the TypeScript compiler happy. Any logic looking for a falsey value probably works just as well when it finds `null`, unless we have something like `if (x===false)` somewhere, which would be `false` if `x` is `null`. I don't think we do and hope we don't have any such logic, but definitely doesn't hurt to keep things more explicit with returning `null` when it's unknown, and `false` when it's known. Either way, the frontend type should match as close as possible the actual potential values returned from the API.

Reply:
Updating to match this from @getvictor 
>`scripts_enabled == null` means this agent is not an orbit agent or it is an older version which is not collecting scripts_enabled